core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -561,9 +578,10 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       val consumerRebalanceListener = new InternalRebalanceListenerForNewConsumer(this, customRebalanceListener)
       if (whitelistOpt.isDefined) {
```

Comment:
Instead of `whitelistOpt.isDefined`, we can write it as:

```scala
whitelistOpt.foreach { whitelist =>
  try {
    consumer.subscribe(Pattern.compile(Whitelist(whitelist).regex), consumerRebalanceListener)
    ...
  }
}
```